Qt 5.7.1 fails to build

This bug seems to be related to https://github.com/Linuxbrew/homebrew-core/issues/146
After the changes made in response to the bug report, installing Qt 5.7.1 should work fine.
But it doesn't...

`brew install qt@5.7 `

```
Downloading https://download.qt.io/official_releases/qt/5.7/5.7.1/single/qt-
Already downloaded: /home/gs/.cache/Homebrew/qt@5.7-5.7.1.tar.xz
==> Downloading https://raw.githubusercontent.com/Homebrew/formula-patches/e8fe6
Already downloaded: /home/gs/.cache/Homebrew/qt@5.7--patch-48ff18be2f4050de7288bddbae7f47e949512ac4bcd126c2f504be2ac701158b.patch
==> Patching
==> Applying restore-pc-files.patch
patching file qtbase/mkspecs/features/qt_module.prf
==> ./configure -verbose -prefix /usr/homebrew/Cellar/qt@5.7/5.7.1 -release -ope
Last 15 lines from /home/gs/.cache/Homebrew/Logs/qt@5.7/01.configure:
xcb-xkb.cpp:55:9: warning: unused variable ‘xcbAtomPrimary’ [-Wunused-variable]
     int xcbAtomPrimary = XCB_ATOM_PRIMARY;
         ^
g++ -Wl,-O1 -o xcb-xkb xcb-xkb.o   -L/usr/homebrew/Cellar/libxcb/1.11.1/lib -lxcb -lxcb-xkb 
xcb-xkb enabled.
checking for xcb-syslibs... 
g++ -c -pipe -O2 -Wall -W -fPIC  -I. -I../../../mkspecs/linux-g++ -o xcb.o xcb.cpp
xcb.cpp:44:27: fatal error: xcb/xcb_icccm.h: Datei oder Verzeichnis nicht gefunden
compilation terminated.
Makefile:183: die Regel für Ziel „xcb.o“ scheiterte
make: *** [xcb.o] Fehler 1
xcb-syslibs disabled.
The test for linking against libxcb and support libraries failed!
 You might need to install dependency packages, or pass -qt-xcb.
 See src/plugins/platforms/xcb/README.
```

So the problem seems to be caused by xcb - again.

I tried to install xcb manually as suggested in the previous bug report.

`brew install xcb-util-wm`

But it didn't fix anything.

--- a/Formula/qt@5.7.rb
+++ b/Formula/qt@5.7.rb
@@ -53,6 +53,18 @@ class QtAT57 < Formula
     sha256 "48ff18be2f4050de7288bddbae7f47e949512ac4bcd126c2f504be2ac701158b"
   end
 
+  unless OS.mac?
+    depends_on :x11
+    depends_on "fontconfig"
+    depends_on "glib"
+    depends_on "icu4c"
+    depends_on "libproxy"
+    depends_on "pulseaudio"
+    depends_on "sqlite"
+    depends_on "systemd"
+    depends_on "libxkbcommon"
+  end
+
   def install
     args = %W[
       -verbose
@@ -65,9 +77,15 @@ class QtAT57 < Formula
       -qt-freetype
       -qt-pcre
       -nomake tests
-      -no-rpath
       -pkg-config
     ]
+
+    if OS.mac?
+      args << "-no-rpath"
+    elsif OS.linux?
+      args << "-qt-xcb"
+      args << "-R#{lib}"
+    end
 
     args << "-nomake" << "examples" if build.without? "examples"
 

